es67x/src/main/scala/tech/beshu/ror/es/IndexLevelActionFilter.scala

```diff
@@ -68,7 +68,8 @@ class IndexLevelActionFilter(nodeName: String,
   private val ror = ReadonlyRest.create(
     new EsIndexJsonContentService(client),
     auditSinkCreator,
-    EsEnv(env.configFile(), env.modulesFile())
+    EsEnv(env.configFile(), env.modulesFile()),
+    rorEsConfig.esNodeConfig,
```

Comment:
In the `IndexLevelActionFilter` class constructor, we have a `nodeName` argument. Let's use EsNodeConfig instead.